### ISSUE TITLE
Report: identify examples that don't have the words

### ISSUE THREAD

**Issue by fcbr** [fcbr/cl-wnbrowser#150]
Since we are using automatic translation for certain examples, we should make sure that the translated examples contain the portuguese words for the synset.  Not sure how to handle this in the UI?  Maybe a report?

**Comment by vcvpaiva** [comment]
indeed, this looks a nice check, that is in principle possible to be automatized. i have been adding the appropriate words introduced by the examples.

**Comment by arademaker** [comment]
This is also related with moving the examples to senses rather than synsets.

**Comment by arademaker** [comment]
1. receber um arquivo com lemas e POS (um par por linha) e devolver uma pagina com o relatório de synsets onde o par ocorre ou se não ocorre.

2. receber um arquivo com texto corrido, rodar freeling e fazer o que 1 acima faz.
